Demisto-sdk release 1.32.5 (#4676)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## 1.32.5 (2024-11-24)
### Breaking
* Removed PA134 from the old validate, the validate ensured the pack has exactly 1 category and that the category is a valid category. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)

### Feature

> * Modified PA103 in the new validate (validate pack categories), the validation will now allow multiple categories for a single pack. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)

Reviewer: ```suggestion
* Modified PA103 validation (validate pack categories). The validation will now allow multiple categories for a single pack. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)
```

> * Modified the configuration section generated by the generate-docs command by removing navigation steps and making the code more generic to support various Cortex platforms. [#4585](https://github.com/demisto/demisto-sdk/pull/4585)

Reviewer: ```suggestion
* Modified the configuration section generated by the generate-docs command by removing navigation steps and generalizing the code to accommodate different Cortex platforms. [#4585](https://github.com/demisto/demisto-sdk/pull/4585)
```

* Added support for non-default registry ports when using custom docker images. i.e: 'registry:5000/repository/image:main' [#4674](https://github.com/demisto/demisto-sdk/pull/4674)

> * Adds support for content_status.json to be used in the content pipeline [#4656](https://github.com/demisto/demisto-sdk/pull/4656)

Reviewer: ```suggestion
*Added support for utilizing content_status.json within content pipelines. [#4656](https://github.com/demisto/demisto-sdk/pull/4656)
```

### Fix

> * Fixed an issue where `pre-commit` inadvertently skipped all files when executed with the `--all-files` flag. [#4668](https://github.com/demisto/demisto-sdk/pull/4668)

Reviewer: ```suggestion
* Fixed an issue where ***demisto-sdk pre-commit*** inadvertently skipped all files when executed with the *--all-files* flag. [#4668](https://github.com/demisto/demisto-sdk/pull/4668)
```

> * Fixed an issue where SDK commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)

Reviewer: ```suggestion
* Fixed an issue where demisto-sdk commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)
```

* Fixed an issue where logging would fail when linking PR to Jira issue in the GitHub workflow step. [#4649](https://github.com/demisto/demisto-sdk/pull/4649)

> * Fixing logger crashes when imported as library. [#4657](https://github.com/demisto/demisto-sdk/pull/4657)

Reviewer: ```suggestion
* Fixed an issue where logger crashed when imported as library. [#4657](https://github.com/demisto/demisto-sdk/pull/4657)
```

* Fixed an issue where logging would fail when creating a Docker image. [#4648](https://github.com/demisto/demisto-sdk/pull/4648)

### Internal
* Moved RN106 and RN107 to the new validate format. These validations ensure there are no missing release notes. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)

> * Fixed an issue where the git statuses of content objects' related files were not calculated correctly. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)

Reviewer: ```suggestion
* Fixed an issue where the git statuses of content objects related files were not calculated correctly. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)
```

## 1.32.4 (2024-11-10)
### Breaking

> * The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.

Reviewer: ```suggestion
* The following commands will be deprecated and removed entirely in the next minor demisto-sdk release: convert, create-content-artifacts, create-id-set, extract-code and lint.
```

### Fix
* Fixed an issue where ***demisto-sdk pre-commit*** inadvertently skipped all files when executed with the *--all-files* flag. [#4668](https://github.com/demisto/demisto-sdk/pull/4668)
* Fixed an issue where demisto-sdk commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)
* Fixed an issue where logging would fail when linking PR to Jira issue in the GitHub workflow step. [#4649](https://github.com/demisto/demisto-sdk/pull/4649)
* Fixed an issue where logger crashed when imported as library. [#4657](https://github.com/demisto/demisto-sdk/pull/4657)
* Fixed an issue where logging would fail when creating a Docker image. [#4648](https://github.com/demisto/demisto-sdk/pull/4648)

### Internal
* Moved RN106 and RN107 to the new validate format. These validations ensure there are no missing release notes. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)
* Fixed an issue where the git statuses of content objects related files were not calculated correctly. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)

## 1.32.4 (2024-11-10)
### Breaking

> * The following commands will be deprecated and removed entirely in the next minor demisto-sdk release: convert, create-content-artifacts, create-id-set, extract-code and lint.

Reviewer: * The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.
```suggestion
* The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.
```
